### PR TITLE
add reset power and toughness option #3402

### PR DESCRIPTION
**Pull request by ebbit1q**
Adds option to reset power and toughness to context menu and keybinds.
See issue #3402

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Make sure to run the `clangify.sh` executable so your builds can be retried for actual failures :)

**Comment by ebbit1q** [comment]
Ok, will do

**Comment by ebbit1q** [comment]
clangify did nothing other than move one file back to otb style while the style guide tells you not to use it?

**Comment by ebbit1q** [comment]
https://github.com/Cockatrice/Cockatrice/blob/master/.github/CONTRIBUTING.md#braces this is literally the opposite of what clang-format does?

File: cockatrice/src/player.cpp [hunk 1]
```diff
@@ -2347,6 +2351,31 @@ void Player::actIncPT(int deltaP, int deltaT)
...
+        Command_SetCardAttr *cmd = new Command_SetCardAttr;
+        QString zoneName = card->getZone()->getName();
+        cmd->set_zone(zoneName.toStdString());
+        cmd->set_card_id(card->getId());
+        cmd->set_attribute(AttrPT);
+        QString ptString = info->getPowTough();
+        cmd->set_attr_value(ptString.toStdString());
+        commandList.append(cmd);
+
+        if (local) {
```
Comment (ZeldaZach) [inline cockatrice/src/player.cpp]: You set playerid at the top for no apparent reason to possibly modify it here. A suggestion could be to ternary case it, or if/else

Comment (ZeldaZach) [reply cockatrice/src/player.cpp]: Also, you're checking if it's a local game each time... can probably do it just once

Comment (ebbit1q) [reply cockatrice/src/player.cpp]: I could but then I'd have to change that everywhere else too.

Comment (ZeldaZach) [reply cockatrice/src/player.cpp]: Ohh.. hmm. Refactor at some point will be necessary, but this is good to go :)

Comment (tooomm) [reply cockatrice/src/player.cpp]: Is there a ticket for that refactor with more details?

**Comment by tooomm** [comment]
> [/.github/CONTRIBUTING.md@`master`#braces](https://github.com/Cockatrice/Cockatrice/blob/master/.github/CONTRIBUTING.md#braces) this is literally the opposite of what clang-format does?

@ebbit1q Can you please open a ticket with the issues you found for further discussion. We can than fix it to make it clear to all other new contributors. Thanks!

**Comment by ebbit1q** [comment]
I opened a ticket: #3413 but there is a lot of other tickets with contributing guide issues

**Comment by tooomm** [comment]
Thanks for the ticket and the fix right after @ebbit1q 👍

**Pull request merged**